podcasts/PodcastListViewController.swift

```diff
@@ -157,6 +157,7 @@ class PodcastListViewController: PCViewController, UIGestureRecognizerDelegate,
             guard let self = self else { return }
 
             self.updateNavigationButtons()
+            self.loadBannerAd()
```

Comment:
If the `subscriptionStatusDidChange` is pushed when the view is open, we should do the banner load to hide/show the banner

Reply:
@bjtitus do you think the cancel task below is enough if the notification is fired while the banner is loading?

Reply:
Good point @bjtitus ! Here [795107b](https://github.com/Automattic/pocket-casts-ios/pull/3523/commits/795107b94afeaeb94f7c0b9cda586a7d20b5c62f)